wetterdienst/provider/dwd/radar/access.py

```diff
@@ -212,6 +218,28 @@ def collect_radar_data(
                         if result.timestamp is None:
                             result.timestamp = date_time
 
+                        if fmt == DwdRadarDataFormat.BUFR:
+                            df = pdbufr.read_bufr(
+                                result.data,
+                                columns=(
+                                    "stationNumber",
+                                    "latitude",
+                                    "longitude",
+                                    "horizontalReflectivity",
+                                ),
```

Comment:
I think more meta data is required to work with radar data. 
1. Number of radial bins 
2. range
3. azimuth

Take a lookt at step 11 https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_load_DWD_opendata_volumes.html

Reply:
Thanks for the hint! This was just a first guess as with pdbufr you will have to set everything you want to read in beforehand and can't just read everything that's being found in the file.

Reply:
Dear @meteoDaniel , could you please check the latest state of the PR? I also couldn't figure out if the information of 
-     Number of radial bins
   -  range
-     azimuth
is contained within the bufr files.